shell/chart/rancher-backup/index.vue

```diff
@@ -113,6 +117,7 @@ export default {
         this.value.s3.enabled = true;
         break;
       default:
+
```

Comment:
another nitpick, but does it need blank line? code is usually right after the `switch` condition

Reply:
updated